parachain/runtime/runtime-tests/src/parachain.rs

```diff
@@ -1,6 +1,8 @@
 mod annuity;
 mod btc_relay;
 mod clients_info;
+#[cfg(not(feature = "with-interlay-runtime"))]
+mod contracts;
```

Comment:
Wondering if we should include this in the interlay runtime since we can also disable those calls. I have anyway included the EVM pallets since I needed to add the runtime APIs for the RPC.